### ISSUE TITLE
❓How does Godot 4.X impact this plugin?

### ISSUE THREAD

**Issue by AdrKacz** [Shin-NiL/Godot-Android-Admob-Plugin#214]
**Godot version: 4.0.2**

**AdMob Plugin version: XXX**

**Issue description:**
Just a question: **How does Godot 4.X impact this plugin?**
Is there work to update the plugin so it uses the latest features of Godot?
Moreover, does this plugin is up to date with the latest AdMob features?

Not sure if the questions are very clear.. I'd be happy to help if any work is needed to *"refresh"* the plugin 😁

Thank you for you work 🎉

**Comment by bonzio** [comment]
> Just minimal conversions here and there, I'm using it regularly on my 4.0.2 projects. No issues at all

What are the conversions that I need to make, are they only about migration to GDScript 2? Like changing export to @export

**Comment by Overvault-64** [comment]
> What are the conversions that I need to make, are they only about migration to GDScript 2? Like changing export to @export

Yeah that and also some minor function renaming or reworking and things like that. Nothing you can't do in a couple hours, depending on the size of your project of course

**Comment by Touff-97** [comment]
If you can read this file, it's fully converted: [admob.txt](https://github.com/user-attachments/files/25186255/admob.txt), just change the extension back to .gd
